### ISSUE TITLE
RA Comments Fab Manual 1

### ISSUE THREAD

**Issue by mailemccann** [AguaClara/horizontal_filtration#40]
Nice job guys!! I'm really excited for what you have planned for the semester. Lets meet next week to discuss my comments if possible.

Abstract-
I think you could better utilize this section to highlight what makes the horizontal filter more optimal in general, rather than talking about previous teams' work. Also you can better outline what your goals for the semester are by phrasing it like "The Spring 2019 team focused on optimizing ..... "

Introduction-
Second sentence: what plant? You haven't mentioned water treatment yet explicitly, so I would specify "water treatment plant" and even better, "modular water treatment plant" when you're talking about the smaller, Oragami Water Lab idea. In the following sentences, you introduce a lot of terms that may be confusing to the reader, including "entrance and exit plates", which I think would be good to expand on a bit. Also I see Sidney already commented this, but I'll reiterate- explain how water flows through the filter and why this design is more optimal!

Previous work- 
Explain why the geometry of the experimental apparatus differs from the actual geometry of the designed system, and why that might cause inaccuracies in testing. Otherwise, this section was well written and easy to follow!

Experimental Apparatus-
The first sentence confused me, is it a typo? If you mean that the pipe was curved due to fabrication errors, explain that fully and how to remedy this. Also, you can be more precise with your language regarding the apparatus. Say exactly how each piece will be attached (you say welded at one point which is good!!), and again, explain why this geometry is more accurate than the previous setup, and maybe explain some of the underlying physical processes that occur when sand fluidizes. This is important in the design, so I think its worth mentioning to the reader. 

Procedure/Fab Details-
Put some type of introduction to these sections to explain what you're about to say. Its a little abrupt going right into a list. Also, keep each listed item gramatically consistent. I would change each point to be instructional rather than what you did. So say "Fill the box with dry sand until both shelves are covered." Instead of "Box filled with dry sand..." @sidneylok can confirm if this okay, but this is the format that I find most concise and readable when writing a Plan of Procedures, Fabrication Methods, etc.

**Comment by mailemccann** [comment]
ALSO!! Drawing looks wonderful. Great use of dimensions and layout. One thing to add would be labels to each part of the drawing eg: TOP, FRONT, SIDE, ENTRANCE PIPE, ENTRANCE PLATE for whatever each view is

**Comment by sidneylok** [comment]
Yes, it would be great to change the points to be instructional for the Procedure section! Great point, Maile.

**Comment by sidneylok** [comment]
Hey guys! Just wanted to add that you should have a bibliography/references section to compile all of citations for the reports and papers you referenced in your lit review and previous work sections. I didn't grade this in your first manual because I'm not sure if it was made clear enough that Fab teams need to include this, but you will be graded on it next time so be sure to include it in your next iteration! Thanks :)
